config/default.yml

```diff
@@ -2699,6 +2699,11 @@ Style/DoubleNegation:
   StyleGuide: '#no-bang-bang'
   Enabled: true
   VersionAdded: '0.19'
+  VersionChanged: '1.0'
```

Comment:
Make this 0.84, please. I'll cut one more release before 1.0 to make sure the `rubocop-ast` separation didn't cause problems.